frameworks/kafka/tests/config.py

```diff
@@ -9,3 +9,8 @@
 DEFAULT_TASK_NAME = 'broker'
 DEFAULT_KAFKA_TIMEOUT = 10 * 60
 DEFAULT_TOPIC_NAME = 'topic1'
+EXPECTED_METRICS = [
```

Comment:
given that this is only used in test_sanity, may as well just put it there and reduce scope